### DISCUSSION TITLE
FP cloud overlapping

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1182] [Help] [answered]
I ask if it exists a parameter of some sort when reading a nod that is showing its level of overlapping with other nodes of the same map (above or below)? If I have a node with cloud in Freeplane (read above) how can I understand that that node is above or below (in terms of cloud overlapping) of another node, which parameter I have to look for? Thanks

**Comment by macmarrum** [comment]
I'm having trouble understanding the issue.
Could you please attach a sample mindmap?

**Comment by Illegit** [comment]
[Cloud overlapping.zip](https://github.com/freeplane/freeplane/files/11585523/Cloud.overlapping.zip)
Here is the map, read the 2 questions inside it. Thankyou

**Reply by dpolivaev** [reply, marked as answer]
If I got your question right, it is about the so-called Z-order defining which node and which cloud is painted at the top and which is painted below if nodes or clouds overlap. The overlap is possible  on your map because you use floating node.

Freeplane does not support defining which node it for overlapping nodes. You can create overlaps using floating nodes or option "View->View settings->Allow compact layout", but there is no support to change the order in which the nodes and clouds are painted.

**Comment by Illegit** [comment]
Yes, Dimitry, you got my point. Thanks for your answer.
